SOPT_Seminar_Task/SOPT_Seminar_Task/Welcome_ViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Welcome_ViewController.swift
+//  Second_Seminar_Task
+//
+//  Created by 곽성준 on 2023/04/14.
+//
+import Then
+import SnapKit
+import UIKit
+
+class Welcome_ViewController: UIViewController {
+    
+    private let tvingImage = UIImageView().then{
+        
+        $0.image = UIImage(named: "image 1.png")
+    }
+    
+    private let idLabel = UILabel().then{
+        
+        $0.text = ""
+        $0.textColor = UIColor(red: 0.839, green: 0.839, blue: 0.839, alpha: 1)
+        $0.font = UIFont(name: "Pretendard-Bold", size: 23)
+        $0.frame = CGRect(x: 0, y: 0, width: 226, height: 74)
+    }
+    
+    private lazy var goToMainButton = UIButton().then{
+        $0.backgroundColor = .red
+        $0.setTitle("메인으로", for: .normal)
+        $0.setTitleColor(.white, for: .normal)
+        $0.addTarget(self, action: #selector(goToMainButtonTapped), for: .touchUpInside)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        style()
+        setLayout()
+    }
```

Comment:
화면을 보면 로그인된 화면에 NavigationBar Back Button 이 있죠?
이건 navigationController 로 처리할 수 있습니다.
이 코드를 추가로 넣으면, 해당 화면에서는 navigationBar 가 사라집니다~
```suggestion
    } override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        self.navigationController?.navigationBar.isHidden = true
    }
```